requests-for-comments/2023-10-27-base3000-launch.md

```diff
@@ -0,0 +1,34 @@
+# Base 3000 & Notebooks launch
+
+We want to launch the ‘first version’ of PostHog 3000 after the next sprint, which means the week starting November 13th (+ 1 week if needed). We have decided to **include Notebooks** in this launch, so that we can have a big launch that combines functionality and design changes.
+
+## Scope for launch
+
+- Dark Mode
+- Base 3000 [Check this mega issue for details](https://github.com/PostHog/posthog/issues/18089)
+    
+<img width="674" alt="Screenshot 2023-10-27 at 10 38 50" src="https://github.com/PostHog/meta/assets/14750837/abb9e559-91ea-408b-a8fd-5105f3c4c626">
+
+## Out of scope for launch (amongst other things)
+
+- Sidebar (Index view)
+- Canvas-powered dashboards
+- Query builder
+- [Check this mega issue for details + the full list](https://github.com/PostHog/posthog/issues/18089)
+
+## Rollout plan
+
+- [x]  Now: Roll out [Notebooks](https://app.posthog.com/feature_flags/8734) to PostHog Team
+- [ ]  Now: Roll out [PostHog 3000 (Base 3000 scope)](https://app.posthog.com/feature_flags/8472) to PostHog Team
```

Comment:
We should specifically ask for feedback after this and people should click through their respective product areas and point out issues to us, so we can help fixing.

Reply:
Yes, 100%. If this timeline looks good, we can share it in All Hands on Monday asking for this, and again a week later, when we have made some more progress on known issues.